Revise desktop wallet docs (#243)
crypto-org-chain/chain-docs: docs/wallets/desktop_wallet.md

If you found your bridge transfer history failed to get latest transferrals, please make sure your Bridge Indexing URL is correct with the value [mentioned above](#_6-1-before-the-bridge-transfer)

## 7. DApp Browser
Since v0.6.4, you’re able to access any DApps available on Cronos Mainnet Beta directly through the Desktop Wallet on both Standard & Ledger Wallet. 
        <img src="./assets/desktop_wallet/7.png" />

### 7.1 - Choose your DApp

> You may go to any DApps you like by either clicking the DApps inside *Popular* / *Saved* section, or type the URL of the DApp directly on the Address Bar at the top. 

Reviewer: ```suggestion
You may go to any DApps you like by either clicking the DApps inside the *Popular* / *Saved* section or typing the URL of the DApp directly on the Address Bar at the top. 
```

### 7.2 - Save your favourite DApp

> You can always save any DApps you like by clicking the bookmark button at the top-right corner. When the bookmark button turns blue, the DApp has been successfully saved into your *Saved* collection.

Reviewer: ```suggestion
You can easily save any DApps you like by clicking the bookmark button at the top-right corner. When the bookmark button turns blue, it means the DApp has been successfully saved into your *Saved* collection.
```

Since v0.6.4, you’re able to access any DApps available on Cronos Mainnet Beta directly through the Desktop Wallet on both Standard & Ledger Wallet. 
        <img src="./assets/desktop_wallet/7.png" />

::: warning DISCLAIMER
You are responsible for your use of any DApps. DApps are controlled solely by their respective project providers. Please check the receiving addresses or contract addresses before you transfer any assets.

> We are not responsible for the accuracy, completeness, or usefulness of such DApps. Accordingly, we neither endorse, recommend, nor give any opinion, advice or whatsoever on such DApps. are not responsible or liable for any losses incurred during your use of DApps.

Reviewer: ```suggestion
We are not responsible for the accuracy, completeness, or usefulness of such DApps. Accordingly, we neither endorse, recommend, nor give any opinion, advice or whatsoever on such DApps, and are not responsible or liable for any losses incurred during your use of DApps.
```
